sdk/schemaregistry/schema-registry/swagger/README.md

```diff
@@ -19,9 +19,7 @@ license-header: MICROSOFT_MIT_NO_VERSION
 output-folder: ../
 source-code-folder-path: ./src/generated
 input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/1e23d91e875e4464e57667639e06408cef99868d/specification/schemaregistry/data-plane/Microsoft.EventHub/preview/2020-09-01-preview/schemaregistry.json
-use-extension:
-  "@autorest/typescript": "6.0.0-dev.20200918.1"
-  "@autorest/modelerfour": "4.15.421"
```

Comment:
It is good we are removing the modelerfour version pin. But, generally we do pin the typescript SDK generator to some specific version. Why is that also removed?

Reply:
So I always regenerate with the latest version of the code generator.

Reply:
This will lead to confusions later. Pin it to a version. You can remove the modelerefour version.

Reply:
I am not following, what would happen to be confused about? I have doing this in all packages I am contributing to and I did not find any issues or got confused! Also, the code gen gets bugfixes all the time and it is important we bring in those fixes in all clients. For instance, this PR gets the agent string fix.